### PR TITLE
Feat:Create Case Study section for android development app

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [x] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [x] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [x] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [x] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [x] Add images, videos, or gifs of UI
- [x] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**

- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook (chrome, firefox, safari)
- [ ] iPhone(safari)
- [x] Available android devices (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

Description:
Create case study section .
**Performance on mobile:
Screenshot from 2023-07-06 16-16-08
performance on desktop:
Screenshot from 2023-07-20 16-23-14

video:

https://github.com/canopas/canopas-website/assets/98794636/c9a8870a-4b62-4316-9171-d66865b3ba29

**Mobile:**
Screenshot from 2023-07-20 16-25-17

Desktop:**
Screenshot from 2023-07-20 16-25-00

### CODE REVIEW

File: vue-frontend/src/components/android-app-development/CaseStudySection.vue [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+            smileplus_800w,
+          ],
+          className: "tw-text-left tw-items-start ",
+          textalign: "tw-right-[5%] tw-w-[52%] tw-ml-[1rem]",
+        },
+      ],
+      cta: [
+        {
+          title: "",
+          content: `"Let’s Start a <span class="tw-text-black-core/[0.87] tw-font-inter-semibold tw-text-[1.375rem] tw-leading-[2.0625rem]">New Project</span> Together"`,
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/CaseStudySection.vue:283]: give some top margin in this content.

File: vue-frontend/src/components/android-app-development/CaseStudySection.vue [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+            disableOnInteraction: false,
+          }"
+          :spaceBetween="20"
+          :navigation="true"
+          class="swiper-container tw-absolute tw-left-0 tw-top-[10px] tw-z-30 tw-mx-auto tw-h-full tw-w-full"
+        >
+          <swiper-slide v-for="item in cases" :key="item">
+            <div
+              class="tw-flex tw-flex-row tw-px-[4rem] tw-py-[6rem] 2xl:tw-p-[8rem] 2xll:tw-p-[11rem]"
+            >
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/CaseStudySection.vue:103]: image and text part not looking good in some devices make image smaller in devices till xl and make image vertically align

File: vue-frontend/src/components/android-app-development/CaseStudySection.vue [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+                        >{{ item.resultcount[1] }}</span
+                      >
+                      <div
+                        class="tw-flex-1 tw-font-inter-regular tw-text-[1rem] lg:tw-text-[1.25rem] tw-leading-[1.625rem]"
+                      >
+                        <span>App Rating</span>
+                      </div>
+                    </div>
+                    <div
+                      class="tw-border-l tw-border-black-core/[0.30] tw-p-[0.8rem]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/android-app-development/CaseStudySection.vue:156]: Check border color in figma

File: vue-frontend/src/components/android-app-development/CaseStudySection.vue [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+            delay: 3000,
+            disableOnInteraction: false,
+          }"
+          :spaceBetween="20"
+          :navigation="true"
+          class="swiper-container tw-absolute tw-left-0 tw-top-[10px] tw-z-30 tw-mx-auto tw-h-full tw-w-full"
+        >
+          <swiper-slide v-for="item in cases" :key="item">
+            <div
+              class="tw-flex tw-flex-row tw-px-[4rem] tw-py-[6rem] 2xl:tw-p-[8rem] 2xll:tw-p-[11rem]"
```
Comment (cp-mansi-d) [inline vue-frontend/src/components/android-app-development/CaseStudySection.vue:102]: give same space of both side (left and right) in md to lg device


https://github.com/canopas/canopas-website/assets/115449373/9e514cb5-c188-4bb7-af9c-fefbd2cebffb

**Review by cp-mansi-d — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**